superset/charts/api.py

```diff
@@ -484,12 +484,7 @@ def get_data_response(
         if result_format == ChartDataResultFormat.CSV:
             # return the first result
             data = result["queries"][0]["data"]
-            return CsvResponse(
-                data,
-                status=200,
-                headers=generate_download_headers("csv"),
-                mimetype="application/csv",
```

Comment:
Do we need to override the default `text/csv` mimetype? The new default seems to be the correct value.